app.py

```diff
@@ -70,4 +81,4 @@ def session_clear(exception=None):
 if __name__ == "__main__":
 
     # Start app
-    app.run(host="0.0.0.0", port=5002)
+    app.run(host="0.0.0.0", port=5000)
```

Comment:
Why move back to 5000? Is something else using 5002? Not important, just curious.